pom.xml

```diff
@@ -7,7 +7,7 @@
 
     <groupId>org.mule.extensions</groupId>
     <artifactId>mule-modules-parent</artifactId>
-    <version>1.1.11-SNAPSHOT</version>
+    <version>1.1.11</version>
```

Comment:
do the proper release in its own separate pr

Reply:
ok, done!